scout/adapter/mongo/hgnc.py

```diff
@@ -357,17 +358,37 @@ def genes_by_alias(self, build=None, genes=None):
 
         return alias_genes
 
-    def ensembl_to_hgnc_mapping(self):
+    def ensembl_to_hgnc_id_mapping(self) -> Dict[str, int]:
         """Return a dictionary with Ensembl ids as keys and hgnc_ids as values
 
         Returns:
-            mapping(dict): {"ENSG00000121410":"A1BG", ...}
+            mapping(dict): {"ENSG00000121410": 5, ...}
         """
         pipeline = [{"$group": {"_id": {"ensembl_id": "$ensembl_id", "hgnc_id": "$hgnc_id"}}}]
         result = self.hgnc_collection.aggregate(pipeline)
         mapping = {res["_id"]["ensembl_id"]: res["_id"]["hgnc_id"] for res in result}
         return mapping
 
+    def hgnc_symbol_ensembl_id_mapping(self) -> Dict[str, str]:
```

Comment:
New function. This gene symbol: ensembl ID is used in parse when PanelApp gene doesn't have Ensembl IDs associated